learning_journal_db/routes.py

```diff
@@ -0,0 +1,6 @@
+def includeme(config):
+    config.add_static_view('static', 'learning_journal_db:static')
+    config.add_route('lists', '/')
+    config.add_route('create', '/journal/new-entry')
+    config.add_route('detail', r'/journal/{id:\d+}')
+    config.add_route('update', r'/journal/{id:\d+}/edit-entry') 
```

Comment:
As a note, not as a deduction, you don't need to turn your routes into raw strings. They're fine as they are. You'll only need to do this in Django